components/shared/SkeletonLoader.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="h-full skeleton-container">
+    <NeoSkeleton
+      class="skeleton-backdrop"
+      rounded
+      border-radius="20px"
```

Comment:
😢

Reply:
? could you elaborate. thanks

Reply:
I thought I asked on a previous PR to unify border radius on skeleton

Reply:
I got no problem if you do so in a follow up